pkg/provider/gitlab/gitlab.go

```diff
@@ -198,19 +202,24 @@ func (v *Provider) CreateStatus(_ context.Context, event *info.Event, statusOpts
 	body := fmt.Sprintf("**%s%s** has %s\n\n%s\n\n<small>Full log available [here](%s)</small>",
 		v.pacInfo.ApplicationName, onPr, statusOpts.Title, statusOpts.Text, detailsURL)
 
-	// in case we have access set the commit status, typically on MR from
-	// another users we won't have it but it would work on push or MR from a
-	// branch on the same repo or if token somehow can have access by other
-	// means.
-	// if we have an error fallback to send a issue comment
 	opt := &gitlab.SetCommitStatusOptions{
 		State:       gitlab.BuildStateValue(statusOpts.Conclusion),
 		Name:        gitlab.Ptr(v.pacInfo.ApplicationName),
 		TargetURL:   gitlab.Ptr(detailsURL),
 		Description: gitlab.Ptr(statusOpts.Title),
 	}
-	//nolint: dogsled
-	_, _, _ = v.Client.Commits.SetCommitStatus(event.SourceProjectID, event.SHA, opt)
+
+	// In case we have access, set the status. Typically, on a Merge Request (MR)
+	// from a fork in an upstream repository, the token needs to have write access
+	// to the fork repository in order to create a status. However, the token set on the
+	// Repository CR usually doesn't have such broad access, preventing from creating
+	// a status comment on it.
+	// This would work on a push or an MR from a branch within the same repo.
+	// Ignoring errors because of the write access issues,
+	if _, _, err := v.Client.Commits.SetCommitStatus(event.SourceProjectID, event.SHA, opt); err != nil {
+		v.eventEmitter.EmitMessage(v.repo, zap.ErrorLevel, "FailedToSetCommitStatus",
+			"cannot set status with the GitLab token because of: "+err.Error())
+	}
```

Comment:
@chmouel @gabemontero handled 
PTAL
Now if there are any error it will show in the events of a user namespace
```
$ oc get events -n test-pipelines | grep FailedToSetCommitStatus
<unknown>   Warning   FailedToSetCommitStatus       repository/savitaashture-test                 cannot set status with the GitLab token because of: POST https://gitlab.com/api/v4/projects/35153477/statuses/48de92146745b56fe5e736c8700dd788e4d66223: 404 {message: 404 References for commit Not Found}
<unknown>   Warning   FailedToSetCommitStatus       repository/savitaashture-test                 cannot set status with the GitLab token because of: POST https://gitlab.com/api/v4/projects/57869138/statuses/48de92146745b56fe5e736c8700dd788e4d66223: 403 {message: 403 Forbidden}

```